hieradata/common.yaml

```diff
@@ -79,3 +79,10 @@ kubernetes::kube_api_advertise_address: "%{::ipaddress}"
 kubernetes::api_server_count: 1
 kubernetes::install_dashboard: true
 kubernetes::controller_address: 169.229.226.75:6443
+kubernetes::nginx_version: '0.21.0'
+kubernetes::workers_ipv4:
+    - 169.229.226.73
+    - 169.229.226.74
```

Comment:
You shouldn't need to hardcode the IPs here, you can look them up with something like this: https://github.com/ocf/puppet/commit/5133f8cf0ecf231bcef204fa1b51b53b6368bc00 (using the `ldap_attr` function since each host has its IP in LDAP)

Reply:
ooooh. yay thanks